### PR TITLE
Update ref name

### PR DESCRIPTION
**Pull request by ilanolkies**
Minor change. Use the same name that is in the figure.

***
<a href="https://pr-preview.s3.amazonaws.com/ilanolkies/vc-data-model/pull/747.html" title="Last updated on Feb 5, 2021, 10:15 AM UTC (bfb5e2e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/747/0e58fb9...ilanolkies:bfb5e2e.html" title="Last updated on Feb 5, 2021, 10:15 AM UTC (bfb5e2e)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
My impression is that the property to be used in the text should be "verifiableCredential", i.e., not "verifiablePresentation" nor "VerifiablePresentation".

**Comment by iherman** [comment]
> My impression is that the property to be used in the text should be "verifiableCredential", i.e., not "verifiablePresentation" nor "VerifiablePresentation".

If I am right, it is probably worth recording this as an official erratum.

**Comment by iherman** [comment]
(Marked the PR as non-substantive, ie, without IPR issues.)

**Comment by ilanolkies** [comment]
Yes, just to clarify, I thought it was referring to this prop

Screen Shot 2020-06-11 at 11 32 03

**Comment by brentzundel** [comment]
Ivan is correct. There is a mistake, but the correction is to replace `verifiablePresentation` with `verifiableCredential` The text refers to this property: 
image

File: index.html [hunk 1]
```diff
@@ -832,7 +832,7 @@ <h3>Presentations</h3>
 information <a>graphs</a>. The first <a>graph</a> expresses the
 <a>verifiable presentation</a> itself, which contains
 <span class="color-text" style="color:#b4a7d6;">presentation metadata</span>.
-The <code>verifiablePresentation</code> property in the <a>graph</a> refers to
+The <code>VerifiablePresentation</code> property in the <a>graph</a> refers to
```
Comment (msporny) [inline index.html:835]:
```suggestion
The <code>verifiableCredential</code> property in the <a>graph</a> refers to
```

@brentzundel is correct.

Comment (ilanolkies) [reply index.html:835]: Actually, I got confused because of the name. That is the correct one!

**Comment by ilanolkies** [comment]
Actually, I got confused because of the name. That is the correct one!

**Comment by OR13** [comment]
@msporny can you leave a comment regarding changes that need to be made to accept this?

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
@brentzundel is correct and it looks like @ilanolkies made the correct change. This PR is ready to merge... but waiting on some process questions I had for @iherman before merging.

**Comment by ilanolkies** [comment]
Hi! Just a friendly reminder of this PR @msporny :)

**Comment by msporny** [comment]
Hey @ilanolkies, apologies for the very long delay. It turns out that your fix kicked off a series of W3C Process questions that are *still*, 8 month later, unresolved. We are having a meeting about closing out those process issues next week.

All that said, your change is Editorial, it's correct, and I'm going to merge it now -- because 8 months for a change like this is ridiculous. :)

The only remaining question is if we mention this in the errata for the current specification, which we will... but that's a separate process.

**Comment by msporny** [comment]
Editorial, multiple positive reviews, no changes requested, no objections, merging.

**Pull request merged**